modules/network-observability-netobserv-cli-install.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+
+// * observability/network_observability/network-observability-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-cli-install_{context}"]
+= Installing the netobserv CLI
+Installing the `netobserv` CLI is a separate procedure from the Operator installation. This means that, even if you have the Network Observability Operator installed from the Operator Hub, you need to install the CLI separately. 
+
+.Prerequisites
+* Install the OpenShift CLI (oc)
+
+.Procedure
+//replace the link with non-github replacement when Julien figures out where to put it.
+. Navigate to the link:https://github.com/netobserv/network-observability-cli/releases/download/v0.0.1/netobserv-cli.tar.gz[netobserv cli tar file].
```

Comment:
:+1: 

I'm still working on CPaaS but will let you know ASAP :smile_cat:

Reply:
@skrthomas @OlivierCazade 

The folder containing the `1.6.0` tar archive should be https://mirror.openshift.com/pub/openshift-v4/clients/netobserv/1.6.0/

We could also point the `latest` version https://mirror.openshift.com/pub/openshift-v4/clients/netobserv/latest/

The full link for `linux` should be https://mirror.openshift.com/pub/openshift-v4/clients/netobserv/1.6.0/netobserv-linux-amd64.tar.gz but I don't think we need to mention it actually. WDYT ?
FYI behind the scene it will resolves to https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/netobserv/1.6.0/netobserv-linux-amd64.tar.gz but we should not reference that one).

Reply:
I added the `latest` version of the link that way we won't need to update this link each new release. 

I dont *think* we need to mention a special link for linux...looking at other CLI doc, they only have one link to the tar file: https://docs.openshift.com/container-platform/4.15/cli_reference/opm/cli-opm-install.html

Reply:
sounds good to me